app/main.py

```diff
@@ -177,17 +161,20 @@
 
 
 @app.callback(Output("page-content", "children"), Input("url", "pathname"))
-def display_page(pathname):
+def display_page(pathname: str):
     """
     This callback takes the url path as input and returns the corresponding page layout,
     thanks to the instantiation functions built in the various .py files.
     """
-    if pathname == "/alert_screen":
+    if len(pathname) == 0 or pathname == "/":
+        return Homepage()
+    elif pathname == "/screen":
         return AlertScreen()
-    elif pathname == "/dashboard_screen":
+    elif pathname == "/device_status":
         return DashboardScreen()
```

Comment:
I understand path renaming here but shouldn't we match function name as well (e.g DashboardScreen) ?

Reply:
Agreed, I'm changing this as well :+1: